integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -95,10 +96,11 @@ public void reregistered(SchedulerDriver driver, Protos.MasterInfo masterInfo) {
 
     @Override
     public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
-      double masterCpu = sConf.getInt(Constants.INTEGRATION_MASTER_RESOURCE_CPU);
-      double masterMem = sConf.getBytes(Constants.INTEGRATION_MASTER_RESOURCE_MEM) / Constants.MB;
-      double workerCpu = sConf.getInt(Constants.INTEGRATION_WORKER_RESOURCE_CPU);
-      double workerMem = sConf.getBytes(Constants.INTEGRATION_WORKER_RESOURCE_MEM) / Constants.MB;
+      // TODO(andrew): Should these be doubles or longs?
```

Comment:
Int/long should be fine. There used to be old logic that required support for fractional values. But that's gone now.